src/app/rundown-overview/components/rundown-overview/rundown-overview.component.scss

```diff
@@ -1,3 +1,20 @@
 a {
   color: var(--blue-color);
+  text-decoration: none;
+
+  &:hover {
+    text-decoration: underline;
+  }
+}
+
+td {
+  height: 55px;
+  border-bottom: solid 1px var(--gray-color);
+  background-color: var(--light-gray-color);
```

Comment:
I must admit, it makes me sad to see the grey background color returning on the RundownOverview table.

Reply:
The only argument was to keep it "close to home" for the first release. But i really want to rethink this general page design and change it to better fit a new Sofie-styling we come up with. Rather than just white pages for the overview and settings and then a completely different looking rundown view as we have it now.

Reply:
That seems like a new task would need to be created for that

Reply:
Well, after some discussion this has been removed anyway!